app/Http/Controllers/BookController.php

```diff
@@ -6,21 +6,37 @@
 use App\Models\Claim;
 use Exception;
 use Illuminate\Http\Request;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
 class BookController extends Controller
 {
-
+    use SoftDeletes;
```

Comment:
You don't need this here - it only needs to go on the model

Reply:
Removed.